Updates for Project Move to Listener

colinfrederickson opened the pull request:
- New HCM Blueprint
- New HCM Listener
- Updates to validations
- Updates to Actions

This will be an intermediate step before moving to the fully new listener functionality

colinfrederickson committed:
Updates for Project Move to Listener

- New HCM Blueprint
- New HCM Listener
- Updates to validations
- Updates to Actions

This will be an intermediate step before moving to the fully new listener functionality

jasonlgong commented:
Testing of listener includes:
- employeeValidations 
- Push to HCMshow 
- reTriggerValidations 
- excelExtractor

colinfrederickson committed:
Updates to Hook Code to get it working

- includes updates to hooks code to get it work - resolves issue with referenceFields

johnmosesman committed:
Update blueprint name
